src/include/daos/object.h

```diff
@@ -116,6 +116,12 @@ enum daos_io_mode {
 #define DAOS_OBJ_REPL_MAX	MAX_NUM_GROUPS
 #define DAOS_OBJ_RESIL_MAX	MAX_NUM_GROUPS
 
+static inline bool
+daos_data_thresh_valid(uint32_t size)
+{
+	return true;
+}
+
```

Comment:
Is this a placeholder for later, or is really any size that fits in a `uint32_t` okay?

Reply:
Yes, just a placeholder and to follow the same flow as how the other properties are managed. Any size should work, but, for instance, we might not want to allow setting a limit higher than the total SCM capacity in the future ... not sure ...